### PR TITLE
update to 1.17.1

### PR DESCRIPTION
**Pull request by cbouss**
cffi 1.17.1

**Destination channel:** main
### Links

- PKG-5760
- https://github.com/python-cffi/cffi
- https://github.com/python-cffi/cffi/releases

### CODE REVIEW

**Review by skupr-anaconda — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -29,11 +29,12 @@ requirements:
     - pip
     - setuptools
     - wheel
+    - libffi {{ libffi }} # [not win]
```
Comment (danpetry) [inline recipe/meta.yaml:32]: worth running the tests? I can't see any compatibility description upstream with libffi. If this is urgent, I won't block (so approved)

**Review by danpetry — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (danpetry) [reply recipe/meta.yaml:32]: I think the static linker performs symbol resolution for shared object dependencies during compile time, so the symbols it needs should be present, at least. And we've got the latest maj.min version of libcffi. so should be fine I think

Comment (cbouss) [reply recipe/meta.yaml:32]: It is urgent (compatibility with latest setuptools), so I'll merge as is.

**Pull request merged**